pages/contacts-call/[id]/index.js

```diff
@@ -1,8 +1,11 @@
 import styled from "styled-components";
 import { callEndSVG } from "../../../components/assets/contactsSVG";
-import { StyledImage } from "../../../components/ContactListItem";
-import { contacts } from "../../../lib/db";
+import { GrayCirclePhotoPlaceholder } from "../../../components/ContactListItem";
+
 import { useRouter } from "next/router";
+import useSWR from "swr";
+
+//--------------------STYLE
```

Comment:
I think this comment is not necessary as the naming of your styled components already indicate that these are stylings, but if this is helpful for you for better orientation it is ok. :)